superset-installer/etc/reflex-provisioner/roles/raf/superset/superset/templates/superset-configmap.yaml.j2

```diff
@@ -7,6 +7,27 @@ data:
     principalKey=`klist -tke /etc/security/keytabs/{{ superset_hive_keytab_name }} | grep '@' | awk -F' ' '{print $4}'|head -1`
     kinit -kt /etc/security/keytabs/{{ superset_hive_keytab_name }} $principalKey
 
+  docker-init.sh: |- 
+    #!/usr/bin/env bash
+
+    set -ex
+
+      echo "Initializing database"
+      superset db upgrade
+
+      echo "Creating default roles and permissions"
+      superset init
+
+    # To start a development web server, use the -d switch
+    
+      gunicorn --bind  0.0.0.0:{{ SUPERSET_CONTAINER_PORT }} \
+          --workers $((2 * {{ SUPERSET_NPROCESSORS }} + 1)) \
+          -k gevent \
+          --timeout {{ SUPERSET_SERVER_WORKER_TIMEOUT }} \
+          --limit-request-line 0 \
+          --limit-request-field_size 0 \
```

Comment:
@ankursinghal2005 do we need to use `limit-request-field` along with `limit-request-field_size`.
Also the link says that it is safe to use to prevent DDOS attacks. It might be vulnerable in production. Please confirm.

http://docs.gunicorn.org/en/stable/settings.html#limit-request-fields

Reply:
Setting this parameter to a very high or unlimited value can open up for DDOS attacks. That's true... We have to identify and R/D what is the max value is suitable for our use cases....

Reply:
Created separate US to track vulnerabilities against DDOS attacks.
https://guavus-jira.atlassian.net/browse/UIC-1182